library/common/build.gradle

```diff
@@ -38,6 +38,9 @@ dependencies {
     testImplementation 'com.squareup.okhttp3:mockwebserver:' + mockWebServerVersion
     testImplementation 'org.robolectric:robolectric:' + robolectricVersion
     testImplementation project(modulePrefix + 'testutils')
+
+    // https://mvnrepository.com/artifact/commons-net/commons-net
```

Comment:
the comment isn't really useful, what about removing it and move under implementation 'androidx.annotation:annotation:' + androidxAnnotationVersion